scripts/imgtool/keys.py

```diff
@@ -0,0 +1,129 @@
+"""
+Cryptographic key management for imgtool.
+"""
+
+from __future__ import print_function
```

Comment:
Wait, this is python, the solution is supposed to be boring: if sys.version[0] < '3'...